sunpy/map/mapbase.py

```diff
@@ -561,19 +604,28 @@ def heliographic_latitude(self):
 
     @property
     def heliographic_longitude(self):
-        """Heliographic longitude"""
+        """
+        Heliographic longitude
+        Meta keyword : hgln_obs
```

Comment:
for cases like this where a a not-None default is set it would be nice to have that in the docstring as well.